### DISCUSSION TITLE
Using JobRunr in production? Tell us about it!

### DISCUSSION THREAD

**Discussion by rdehuyss** [jobrunr/jobrunr#146] [Show and tell]
## 👋 Hi there - this is Ronald, the creator of JobRunr...

First of all, thanks sincerely for using and supporting JobRunr. I will try my best to make JobRunr better, and keep the community and eco-system growing. You could help me too by telling me that you are using JobRunr in your project / product.

Please submit a comment in this discussion and include the following information:

- your company, school or organization.
- your city and country.
- your contact info: blog, email, twitter (at least one).
- for what business scenario do you use JobRunr

You can refer to the following sample answer for the format:

```
* Organization: Rosoco
* Location: Leuven, Belgium
* Contact: ronald.dehuysser@gmail.com
* Purpose: used as cornerstone in a distributed service architecture for generating a large amount of documents.
```
Thanks again for your participation !

**Comment by sa1nt** [comment]
* Organization: CTCo
* Location: Riga, Latvia
* Contact: anatolijs.mozgovs@gmail.com
* Purpose: In the project I work on, we use it as a replacement to Spring `@Scheduled` annotation for running scheduled jobs, and the main advantage for us is that it's easy to change the schedule without redeploying the application.
Our first and most obvious option was Quartz, but it doesn't play nearly as nice with a NoSQL data store, which is another huge advantage of JobRunr for us. 
And then, for our use cases JobRunr is orders of magnitude easier to set up and maintain than Quartz, which is also a great feat.

**Comment by jakub-hirnsal** [comment]
* Organization: Bisnode / Dun & Bradstreet
* Location: Team in Czech Republic
* Contact: 
* Purpose: JobRunr is used for scheduling large exports into PDF and XLS. User can trigger export in a Web application. Job is run in background and progress is shown in the app. Once it's done, use can download result.

**Comment by rafaeltmiranda** [comment]
* Organization: Mindera - outsourcing to TVG / Fanduel
* Location: Mindera - Portugal | TVG/Fanduel - USA
* Contact: rafael.miranda@mindera.com
* Purpose: We are in the middle of a project to bring TVG's product into Fanduel's ecosystem. The stream I'm working on aims to migrate all users into the new product. This is a significant endeavor, as there are a lot of moving pieces, things that we need to move from one platform to another, per user, during the migration. JobRunr is being used, with some tweaks, to handle the migration in a Step by step (job by job) manner. So basically the process is triggered by calling a REST endpoint which enqueues the first step for each user. Then each step decides what step to enqueue next. It makes what could be very complex into a very easy architecture, simple to understand and adapt for future migrations. It is not yet in production. We are in the testing phase. But it will be soon. I'm sure this is an unusual use case, and it will be running at full power for a couple of hours and will be scaled down in the end. But I think it is worth sharing :)
